### PR TITLE
quassel: update to 0.12.5. 

### PR DESCRIPTION
**Pull request by Piraty**
client works,
core works

### CODE REVIEW

File: srcpkgs/quassel/template [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
 maintainer="Toyam Cox <Vaelatern@gmail.com>"
 hostmakedepends="pkg-config"
-makedepends="zlib-devel qt-devel qt-webkit-devel libdbusmenu-qt-devel phonon-devel qca-devel"
+makedepends="qt5-tools-devel qt5-webkit-devel qt5-plugin-sqlite qt5-plugin-mysql
+ qt5-plugin-odbc qt5-plugin-pgsql libdbusmenu-qt5-devel qt5-plugin-tds
+ qt5-script-devel phonon-qt5-devel qca-qt5-devel zlib-devel"
 _common_deps="desktop-file-utils"
-depends="${_common_deps} qt-plugin-sqlite quassel-client-shared>=${version}_${revision}"
+depends="${_common_deps} qt5-plugin-sqlite quassel-client-shared>=${version}_${revision}"
 license="GPL-3"
```
Comment (maxice8) [inline srcpkgs/quassel/template]: please use spdx identifier

**Comment by Piraty** [comment]
done, thanks

File: srcpkgs/quassel/template [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
-makedepends="zlib-devel qt-devel qt-webkit-devel libdbusmenu-qt-devel phonon-devel qca-devel"
+makedepends="qt5-tools-devel qt5-webkit-devel qt5-plugin-sqlite qt5-plugin-mysql
+ qt5-plugin-odbc qt5-plugin-pgsql libdbusmenu-qt5-devel qt5-plugin-tds
+ qt5-script-devel phonon-qt5-devel qca-qt5-devel zlib-devel"
 _common_deps="desktop-file-utils"
-depends="${_common_deps} qt-plugin-sqlite quassel-client-shared>=${version}_${revision}"
-license="GPL-3"
-configure_args="-DEMBED_DATA=1 -DWANT_QTCLIENT=ON -DWANT_MONO=ON -DWANT_CORE=ON -DWITH_OPENSSL=ON -DWITH_DBUS=ON -DWITH_PHONON=ON -DWITH_WEBKIT=ON"
+depends="${_common_deps} qt5-plugin-sqlite quassel-client-shared>=${version}_${revision}"
+license="GPL-2.0, GPL-3.0"
```
Comment (maxice8) [inline srcpkgs/quassel/template]: that is deprecated SPDX identifier, GNU licenses use -or-later or -only

Comment (Vaelatern) [reply srcpkgs/quassel/template]: It's neither. It's under both.

File: srcpkgs/quassel/template [hunk 3]
```diff
@@ -1,21 +1,23 @@
...
+configure_args="-DEMBED_DATA=1 -DWANT_QTCLIENT=ON -DWANT_MONO=ON -DWANT_CORE=ON -DWITH_WEBKIT=ON -DUSE_QT5=ON"
 system_accounts="quassel"
-homepage="http://www.quassel-irc.org"
-distfiles="http://quassel-irc.org/pub/quassel-${version}.tar.bz2"
-checksum=93e4e54cb3743cbe2e5684c2fcba94fd2bc2cd739f7672dee14341b49c29444d
+homepage="https://www.quassel-irc.org"
+distfiles="https://github.com/quassel/quassel/archive/${version}.tar.gz"
```
Comment (Vaelatern) [inline srcpkgs/quassel/template]: Please put this back.

Comment (Piraty) [reply srcpkgs/quassel/template]: alright, reset to original (but https instead)

**Comment by Vaelatern** [comment]
One last change I want:
```
* WebEngine support for URL previews (deprecates WebKit support with Qt 5)
* Disable WebKit by default due to it being unmaintained and a security risk
```
Thank you for this work.

**Comment by Piraty** [comment]
thanks @Vaelatern.
I merged the `-shared` subpkg into `-client-shared` as it only contained icons...
```
$ xbps-query -Rf quassel-shared
/usr/share/icons/hicolor/48x48/apps/quassel.png
/usr/share/pixmaps/quassel.png
```
Also, `desktop-utils` is only needed for the desktop related subpackages and was moved to the according subpackage.

xbps doesn't pick up the sqlite runtime dependency of the core on it's own, so it stays in `depends`
I tested the core and the client, both work

ready for next round of review.

**Comment by Piraty** [comment]
moved: https://github.com/void-linux/void-packages/pull/4

**Pull request closed without merging**